src/components/Blog/Feed/Item/index.tsx

```diff
@@ -67,7 +67,7 @@ const Item: React.FC<IBlogFeedItemProps> = ({
     >
       <Link href={slug} className={styles.pictureLink}>
         {picture ? (
-          <Image fluid={image} className={styles.picture} />
+          image && <Image fluid={image} className={styles.picture} />
         ) : (
           <Placeholder className={styles.picture} />
         )}
```

Comment:
If I understand correctly, I think this should be:

```jsx
{image ? (
  <Image fluid={image} className={styles.picture} />
) : (
  <Placeholder className={styles.picture}
)}
```

because we should show a placeholder if there's no `image`, and with this we'll just show nothing if there's `picture` but no `image`- that said, I'm not sure this could actually happen since `image` and `picture` are practically the same thing, so it's more of a stylistic nitpick.